sljit_src/sljitNativeX86_common.c

```diff
@@ -484,16 +476,17 @@ static void get_cpu_features(void)
 {
 	sljit_u32 feature_list = CPU_FEATURE_DETECTED;
 	sljit_u32 info[4];
+	sljit_u32 *pinfo = info;
```

Comment:
info is pointer, why do you need an alias?

Reply:
the info "label" is a pointer, but `info` is really an array and you need a pointer to it for the assembler to use the correct type on its options.